src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter format = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
make this constant (static + naming)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names